dev/import-beats/kibana.go

```diff
@@ -234,29 +234,40 @@ func convertToKibanaObjects(dashboardFile []byte, moduleName string, dataStreamN
 			return nil, errors.Wrapf(err, "retrieving type failed")
 		}
 
+		id, err := object.getValue("id")
+		if err != nil {
+			return nil, errors.Wrapf(err, "retrieving id failed")
+		}
+
+		dashboardName := id.(string)
+		if aType == "dashboard" && !strings.HasPrefix(dashboardName, moduleName+"-") {
+			dashboardName = moduleName + "-" + dashboardName
+		}
+
+		_, err = object.put("id", dashboardName)
+		if err != nil {
+			return nil, errors.Wrapf(err, "putting new ID failed")
+		}
+
 		data, err := json.MarshalIndent(object, "", "    ")
 		if err != nil {
 			return nil, errors.Wrapf(err, "marshalling object failed")
 		}
 
-		data = replaceBlacklistedWords(
-			replaceFieldEventDataStreamWithStreamDataStream(
-				data))
+		data = prependModuleNameToDashboard(replaceBlacklistedWords(
```

Comment:
The `prependModuleNameToDashboard` function name confused me for a bit since we had already done that in the new code above. I had to read the function body to realize it was about updating the dashboard _links_. Consider renaming the function to be more explicit about what it does?

```suggestion
		data = prependModuleNameToDashboardLinks(replaceBlacklistedWords(
```

Reply:
You're right here. Rephrased.